core/src/volume.rs

```diff
@@ -0,0 +1,40 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::Path;
+
+#[derive(Clone, Copy, Debug)]
+pub struct Volume {
+    pub volume: u16,
+}
+
+impl Volume {
```

Comment:
Why not just write the raw 0..0xFFFF value to the file? This isn't meant to be modified by other programs, so it's simpler that way.

Reply:
That was originally to mirror `--initial-volume`, which is represented as a percent. Given that users never need to access the cached value, I can remove that conversion.